jest.setup.ts

```diff
@@ -2,6 +2,9 @@ import 'regenerator-runtime/runtime';
 import '@testing-library/jest-dom';
 import 'jest-canvas-mock';
 import './__mocks__/dh-core';
+import Log from '@deephaven/log';
+
+Log.setLogLevel(-1);
```

Comment:
Tests should always have the env variables, no? It's sometimes useful to enable logs in tests, and this would negate that. Should read REACT_APP_LOG_LEVEL ohhhh wait that's only in code-studio. Hm.
Either way, should allow it to read from an env var, default to -1 if not provided... so then you could do something like `DEEPHAVEN_LOG_LEVEL=4 npm test` or something.

```
let logLevel = parseInt(process.env.DEEPHAVEN_LOG_LEVEL ?? '', 10);
if (!Number.isFinite(logLevel)) {
  logLevel = -1;
}
Log.setLogLevel(logLevel);
```

Reply:
Yep the env variables aren't read since tests are run from root. I'll add an env variable